datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/assertion/EntityAssertionsResolver.java

```diff
@@ -84,6 +92,22 @@ public CompletableFuture<EntityAssertionsResult> get(DataFetchingEnvironment env
                 gmsResults.stream()
                     .filter(Objects::nonNull)
                     .map(r -> AssertionMapper.map(context, r))
+                    .filter(
```

Comment:
tiny nit: i'd prefer to see this nested logic extracted into private method here:

.filter(this::assertionExists)